subtitles.go

```diff
@@ -230,15 +282,20 @@ type StyleAttributes struct {
 	WebVTTVertical       string
 	WebVTTViewportAnchor string
 	WebVTTWidth          string
+	WebVTTColor          string
 }
 
 func (sa *StyleAttributes) propagateSSAAttributes() {}
 
-func (sa *StyleAttributes) propagateSTLAttributes() {}
+func (sa *StyleAttributes) propagateSTLAttributes() {
+	sa.WebVTTLine = sa.WebVTTLineFromSTL()
```

Comment:
Check if `STLJustificationCode` is not `nil` before calling `WebVTTLine` instead:

```go
if sa.STLJustificationCode != nil {
    sa.WebVTTLine = sa.STLJustificationCode.WebVTTLine()
}
```